File: 1929-concatenation-of-array/1929-concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums)
    {
        int []arr=new int[nums.length+nums.length];
        int j=0;
        for(int i=0;i<nums.length;i++)
        {
            arr[i]=nums[j];
            arr[i+nums.length]=nums[j];
            j++;
        }
        return arr;
    }
}

File: 1929-concatenation-of-array/1929-concatenation-of-array.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        l=len(nums)
        arr= []
        for n in nums:
            arr.append(n)
        for n in nums:
            arr.append(n)
        # print(arr)
        return arr
